AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/SQLStatement+Select.swift

```diff
@@ -22,6 +22,7 @@ struct SelectStatementMetadata {
     // TODO remove additionalStatements once sorting support is added to DataStore
```

Comment:
since sorting is now supported, we should address this `TODO`: `additionalStatements` can be safely replaced with sort

Reply:
removed this line